resources/user-guides/attach-data.md

```diff
@@ -155,25 +155,26 @@ If you do not have existing rules for this property, the **Create New Rule** but
 4. On the right pane, in the **JSON Payload** field, type the data that will be added to this event.
 5. Click **Keep Changes**.
 
-On the right pane, you can add a freeform JSON payload that adds data to an SDK event before an extension that is listening for this event can hear the event. In this example, additional profile parameters are added to this event before the Target extension processes it. The added profile parameters will now be added on outgoing Target retrieve location content requests.
+On the right pane, you can add a freeform JSON payload that adds data to an SDK event before an extension that is listening for this event can hear the event. In this example, a custom mbox with custom mbox parameters is added to the event before the Target extension processes it. The added custom mbox will now be added on outgoing Target retrieve location requests.
 
-In the following example, **extraKey**, **extraKey2**, and a data element defined for the **OS version** are added to the profile parameters of the Target event. Values for the new keys can either be hardcoded in the rule, or dynamically determined by the SDK when this event processes by using Data Elements.
+In the following example, an mbox named **custom_mbox** with an additional mbox parameter named **customparams** is added to the Target event.
 
 ![](../../.gitbook/assets/target-attach-data-json-example.png)
 
-The following example shows how the data element for OS version was created.
-
-![](../../.gitbook/assets/target-attach-data-data-element-setup.png)
-
-In addition to being able to add custom profile parameters to each Target retrieve location request, custom mboxes can be added as well. The following example contains a valid JSON payload for this use case:
+In the above example, the JSON payload adds custom mbox parameters only for the custom mbox added. Custom parameters can be added to each of the Target retrieve location objects instead. The following example contains a valid JSON payload for this use case:
```

Comment:
Please reword as follows:
"In the above example, the JSON payload adds custom mbox parameters only for the custom mbox that was added. You can also add custom parameters to each of the Target retrieve location objects. The following example contains a valid JSON payload for this use case:"